app/src/main/aidl/net/programmierecke/radiodroid2/IPlayerService.aidl

```diff
@@ -23,6 +24,7 @@ StreamLiveInfo getMetadataLive();
 ShoutcastInfo getShoutcastInfo();
 MediaSessionCompat.Token getMediaSessionToken();
 boolean isPlaying();
+boolean hasPlaybackStarted();
```

Comment:
Such name unfortunately will only add confusion. Would be better to return `PlayState`.